src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name,
+    age,
+    sex,
+    isMarried,
+    partnerName } = person;
+  const partner = sex === 'm'
```

Comment:
partner looks like object with partner

```suggestion
  const partnerType = sex === 'm'
```